remo/base/views.py

```diff
@@ -38,14 +39,23 @@ def login_failure(self, msg=''):
         return super(OIDCCallbackView, self).login_failure()
 
 
-@cache_control(private=True, no_cache=True)
+@cache_control(private=True, max_age=60 * 60 * 8)
```

Comment:
I explicitly said to cache the content. By caching the view we have some side effects like, featured rep might not show up. Also this decorator adds ``cache-control`` headers which instruct the user agent to cache stuff. That means that for all users loading the page is going to be slow for the first time.